src/Parser/Paladin/Protection/Modules/Features/ShieldOfTheRighteous.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { formatPercentage, formatThousands } from 'common/format';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import MAGIC_SCHOOLS from 'common/MAGIC_SCHOOLS';
+
+
+class ShieldOfTheRighteous extends Analyzer {
+
+  physicalHitsWithShieldOfTheRighteous = 0;
+  physicalDamageWithShieldOfTheRighteous = 0;
+  physicalHitsWithoutShieldOfTheRighteous = 0;
+  physicalDamageWithoutShieldOfTheRighteous = 0;
+
+  on_toPlayer_damage(event) {
+    if (event.ability.type === MAGIC_SCHOOLS.ids.PHYSICAL && this.selectedCombatant.hasBuff(SPELLS.SHIELD_OF_THE_RIGHTEOUS_BUFF.id)) {
+      this.physicalHitsWithShieldOfTheRighteous += 1;
+      this.physicalDamageWithShieldOfTheRighteous += event.amount + (event.absorbed || 0) + (event.overkill || 0);
```

Comment:
Might make sense to calculate the damage pre SotR mitigation and use that for calculating the efficiency instead (if you take 100 total dmg pre mitigation and mitigate half of it (and let's assume it's 50% DR), I think showing 50/100 = 50% mitigated would be more intuitive than 25mitigated / (50 unmit + 25 mitigated) = 33%. Might not be possible to calculate though because of the armor changes in BFA, I haven't though it through.

Reply:
Just checked it, SotRs DR is just added to the armor-field in WCLs API.
Could probably track back the amount of armor provided by SotR (since its 150% of Str), but that doesn't only require an accurate str tracking but also the proper math behind the armor (DR and armor cap and whatever).

Reply:
Was there also something about harder bosses (as in later raid tiers) requiring more armor to get the same damage reduction in BFA? Or maybe that was just Stagger working like that, I can't remember. Anyway it's probably a thing for the future.